File: problemas/1100/1151.js
const n = Number(require('fs').readFileSync('/dev/stdin', 'utf8'));
let sequencia = [];
sequencia.push(0);

if (n > 1) {
  sequencia.push(1);
}

for (let i = 2; i < n; i++) {
  sequencia.push(sequencia[i - 1] + sequencia[i - 2]);
}

console.log(sequencia.join(' '));

File: problemas/1100/1151.dart
import 'dart:io';

void main() {
  final n = int.parse(stdin.readLineSync());
  var sequencia = new List<int>();
  sequencia.add(0);

  if (n > 1) {
    sequencia.add(1);
  }

  for (var i = 2; i < n; i++) {
    sequencia.add(sequencia[i - 1] + sequencia[i - 2]);
  }

  stdout.writeln(sequencia.join(' '));
}